package.json

```diff
@@ -1,61 +1,64 @@
 {
   "name": "@metamask/detect-provider",
   "version": "2.0.0",
-  "description": "A tiny utility for detecting the MetaMask Ethereum provider, or any EIP 1193-compliant provider.",
```

Comment:
There should be no functional changes to this file. The Node version should be the same as well as the set of published files. I did change `typings` to `types` but that should work the same.